.github/workflows/ci.yml

```diff
@@ -56,6 +56,14 @@ jobs:
       - name: Run code style check
         run: yarn style
 
+  changes-in-latest-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: checks modified files
+        run: yarn changes-in-version
```

Comment:
You need to add another step in here to install the dependencies like on line 53 above.